tuftool/src/main.rs

```diff
@@ -77,14 +78,16 @@ impl Program {
 enum Command {
     /// Create a TUF repository
     Create(create::CreateArgs),
-    /// Download a TUF repository's resources
+    /// Download a TUF repository's targets
     Download(download::DownloadArgs),
     /// Update a TUF repository's metadata and optionally add targets
     Update(Box<update::UpdateArgs>),
     /// Manipulate a root.json metadata file
     Root(root::Command),
     /// Delegation Commands
     Delegation(Delegation),
+    /// Clone a fully functional TUF repository
```

Comment:
```suggestion
    /// Clone a TUF repository, including metadata and some or all targets
```
It may not be fully functional depending on their given arguments :)